Formula/w/whatweb.rb

```diff
@@ -0,0 +1,76 @@
+class Whatweb < Formula
+  desc "Web scanner. Identify the technology stack that powers a website"
+  homepage "https://morningstarsecurity.com/research/whatweb"
+  url "https://github.com/urbanadventurer/WhatWeb/archive/refs/tags/v0.5.5.tar.gz"
+  sha256 "96dedb6a377184fb8f5fd3f2a81c26ff8c92c4dc1503ce409793a1e7ab23695d"
+  license "GPL-2.0-only"
+  head "https://github.com/urbanadventurer/WhatWeb.git", branch: "master"
+
+  uses_from_macos "ruby" => :build, since: :ventura
+
+  def install
+    ENV["GEM_HOME"] = buildpath/"gem_home"
+    system "gem", "install", "bundler"
+    ENV.prepend_path "PATH", buildpath/"gem_home/bin"
+    system "bundle", "install"
+    system "make", "install",
+      "prefix=#{prefix}",
+      "-e", "BINPATH=#{bin}",
+      "-e", "DOCPATH=#{doc}",
+      "-e", "LIBPATH=#{lib}",
+      "-e", "MANPATH=#{man}"
+
+    # gem 'addressable', '~> 2.8', '>= 2.8.5'
+    resource "addressable" do
```

Comment:
Put the `resource` blocks outside of `def install`. Other maintainers may agree but: if this has a `Gemfile.lock` it's using, I'm not convinced you need to use a `resource` for all of these instead of just using `bundle install`.

Reply:
Previous iterations of the formulae added gems after the bundler run. That's why I suggested resources.

Reply:
Yeh, I'm pretty confused as to why these gems would be manually installed like this rather than using `system "bundle", "install", "--local", "--path=#{libexec}/vendor"` or something (like other Ruby formulae do).